<p>
  Stateless flag should be to true 
</p>  

<h2>Noncompliant Code Example</h2>
<pre>
  Any service with a branch step that doesn't have the label expression set to 'true' while no switch value has been given.
</pre>

<h2>Compliant Solution</h2>
<img style="width: 200px;" src="data:image/png;base64, iVBORw0KGgoAAAANSUhEUgAAAhQAAABnCAIAAADXMUz0AAAAAXNSR0IArs4c6QAAAARnQU1BAACxjwv8YQUAAAAJcEhZcwAAJOgAACToAYJjBRwAAAhLSURBVHhe7d1dqGVlHcdxRdTU8j18waIUsyBB6kIE04ouQpHCSJBSyquKkEyiQCQVQQiKJIiyLiIrKLqIkCJUKO2FjKIuMiVQTMNRMTMtLdQpvr/nYstm7dY+8z9z9sz/+8GL8VnPWmf2uNb6ztnPPssDdktSqV27Hv3fP+NftJ8yHpKKGY8OjIekYsajA+MhqZjx6MB4SCpmPDowHpKKGY8OjIekYsajA+MhqZjx6MB4SCpmPDowHpKKGY8OjIekYsajA+MhqZjx6MB4SCpmPDrYiHicjwOWnIAP4ymMHTbS0zgFP8HYIDVjPDowHmWMhxTGo4MNiscNeA7P4l6ciyswdthR1+BCjCG8iC/hUYwNUjPGowPjsTbjIa1mPDqYjMdfcc+WZN9xoBkSj5swhhb8AK/GGNpRU/GQFMajA+OxNuMhrWY8OpiMxyP4Ij4x2xfwMMaBZpgTj+ORkbwplEX1B5HxuBWnYQzhKFyNN+IVyK/vwpg64RIcgoOQY16MzMnI3VgcuRKvwyuR/PwOH0QCGZ9HjhCP41KciJNxFZ7HmCrtKOPRgfEwHlIx49HBZDxews/xcVy00seQW3D2HQeaYSoeu3AecovM+NbicQQSuT/gH/gI3owxdaXVb1slFYvxyNdNtB5AwnwmkpN8tPcZfA8H4iHkzzN/DolHZj6Gs3Aj8hWlnWU8OjAexkMqZjw6mIxH5La1OiF7ko1IPA7DcTgaGcnt/j/I/D2Jx+0YQ7gTB2MMrbS1eCx/3eTwvRhDyJ9hfj/5vd2HvN4nMKbiZpyDMSTtKOPRgfEwHlIx49HB/4lH5Ha2nJA9z0YkHp9BFtu/iWOQkTEVtfHIjT4L4GNopap4fBrvwRhasLjXHcjrzSNbFh2LvAk2dpZ2lPHowHgYD6mY8ehgVjwieUhCPoufIeNj0pYkHssL5nkk4tvxAjKeheLcTO9HxmPvxOMCjKEF2xGPvMYsoefBLWOStJGMRwfGw3hIxYxHB2vEI/L4v9zO8uuxYQ9MxSMfpX09Frfm62Zp/TrklpqHo1yO7YvH9TgdWb5+Etm6HfHI680DIvPq8nWT7XzwN/9Fxs7SjjIeHRgP4yEVMx4drB2P7TAVj/gFDsWvkfHbcCoOx9m4FtsXjzwoPsc/Evn4QLZuRzwyko8JXIaTkA80n4FbkJnSzjIeHRgP4yEVMx4dbEQ8JO1PjEcHxkNSMePRgfGQVMx4dGA8JBUzHh0YD0nFjEcHxkNSMePRgfGQVMx4dGA8JBUzHh0YD0nFjEcHxkNSMePRgfGQVMx4dGA8JBUzHh0YD0nFjEcHxkNSMePRgfGQVMx4dGA8JBUzHh0YD0nFjEcHxkNSMePRgfGQVMx4dGA8JBUzHh0YD0nFjEcHxkNSMePRgfGQVMx4dGA8JBUzHh0YD0nFjEcHxkNSMePRgfGQVMx4dGA8JBUzHh0YD0nFjEcHxkNSMePRgfHYh92EP2MMSRvAeHRgPPZhxkObyXh00C4ef8NHcQoOQX79PtyHscMGOwo/xhiSNsDqeJyGA2b7E8bO2hjGw3hIxYxHB+3icQHOw2/xFP6IvBH0GMYOuAYXYgxtSdVxwnhoM62Ox/N4bsENeBvG0IIXMXbWxjAexkMqZjw6aBSPnIKH4k6MDTNU3fSrjhPGQ5tpdTyW5S9t52MMaeMZj1mqbvpVxwnjoc1kPDpo97bV2Xg/nsHYMOESZFH9IOSWfTEy52q8FkfiVHwLmTPnOI/jUpyIk3EV8s1+ZkaOsByPdY/zVbwBRyCv5XMYkzB/pjrbjngchzvwVhyMfLwlS+sPYuyAW5El+jGEda8RLTMexsN4qJjx6KBdPB7Au5Gb75VYPu0WrX676R48iZfwDeQtsqeRmVPHyV5Zxs8JnbBl6f4s3IixA/L7X4zHusf5C3Lh/RD/xt9xP9adKW1HPPKXlXfg98iV9ShyZi5fxcvxWPca0RTjYTyMh4oZjw7axWPRL5EfDDwM+eHBf2FMwtRNf0pOx5zQOdEzPnWcxW+9n8DYgJtxDsYQluOx7nF2Id/+fxlT37DPnyltXzxuxxjCuvFY9xrRFONhPIyHihmPDlrHY1FOuzfhUxgbMHXTz7fA38W7cBKybJ4T9DfI/KnjZBkw809YcizOxNgBy/HY2nF+irwhcDQ+iVyWYxLmz1RnmxyPrV0jWmY8hpx2xmN1EubPVGfGowPj8TLXIhkYQ8hNP482GUP4FXITvxt52El+IDEn6HI8lo+TBecD8SzGhpXydX+EjGztOIuyMP4BvAVjw5L5M9XN3oxHFrpzreX8HxuwHI/M2ZNrRGE8XsZ4zE/C/Jnqxnh00CgeWQb/EG7DQ8jJ9x3krZivY+yG63E6ssiWD+Z+G8fjYeRRbt9HTujFeEwdJ7E5F5cjW/O22CPISZ/jRN4i+woyc93jvIC7kAXwfyIPqnsN1p0p7c145JzPjxBehyQhH6DPVbAYj8yff41oivEwHsZDxYxHB43ikR8pyumVxxscg1chjy1JQsYOC+5FTsEshl+E/KDcFch4HteRt7/yP5hajMfUcbI1S3+XIWFIzM7ALcjM+BryKrJwnfH5x8nMdyILhocjb0MtPj5y/kxpb8Yj8tfBPBYoZ2au6FyJueLGVOR8nnONaIrxMB7GQ8WMRwcumEsqtm48tC8yHpKKGY8OjIekYsajA+MhqZjx6MB4SCpmPDowHpKKGY8OjIekYsajA+MhqZjx6MB4SCpmPDowHpKKGY8OjIekYsajA+MhqZjx6MB4SCpmPDowHpKKGY8OjIekYsajA+MhqZjx6MB4SCpmPDowHpKKGY/93+7d/wVTmL35kH3nGwAAAABJRU5ErkJggg=="/>