<p>
  Services with interface element 'EXIT' need to be configured correctly.
</p>

<pre>
  Services with interface element 'EXIT' step need to be configured correctly:
  	- The "Exit from " property needs to be configured.
  	- The failure message needs to be added if the "Signal" property is set to "Failure".
</pre>

<h2>Noncompliant Code Example</h2>
<pre>
Any non-compliant variant
</pre>

<h2>Compliant Solution</h2>
<pre>
	- Configure the "Exit from " property.
  	- Add the failure message if the "Signal" property is set to "Failure".
</pre>