<p>
  All branch steps should be configured correctly
</p>

<p>
  All branch steps should have the switch value set or have the label expressions set to 'true'
</p>

<h2>Noncompliant Code Example</h2>
<pre>
  Any service with a branch step that doesn't have the label expression set to 'true' while no switch value has been given.
</pre>

<h2>Compliant Solution</h2>
<pre>
  Give a switch value or set the label expression value to 'true'.
</pre>