<p>
  Services should not contain any disabled elements.
</p>

<p>
  Services should be free of any disabled code snippets.
</p>

<h2>Noncompliant Code Example</h2>
<pre>
Any disabled code.
</pre>

<h2>Compliant Solution</h2>
<pre>
No disabled code.
</pre>
