<p>
  Empty services should be removed.
</p>

<p>
  Packages should be free of any empty services.
</p>

<h2>Noncompliant Code Example</h2>
<pre>
  Any package with a service that doesn't have any flow steps.
</pre>

<h2>Compliant Solution</h2>
<pre>
  Remove all services that don't have any flow steps or adjust the service.
</pre>