<p>
  Empty 'MAP' interface elements should be removed from the service.
</p>

<p>
  Services should be free of any empty 'MAP' interface elements.
</p>

<h2>Noncompliant Code Example</h2>
<pre>
Any service with a 'MAP' interface element that doesn't have any influence.
</pre>

<h2>Compliant Solution</h2>
<pre>
  Remove all 'MAP' interface element that doesn't have any influence.
</pre>