<p>
  Interface elements should contain comments.
</p>

<p>
  Interface elements should contain comments. This allows for correct creation of documentation.
</p>

<h2>Noncompliant Code Example</h2>
<pre>
Any interface element without comment.
</pre>

<h2>Compliant Solution</h2>
<pre>
The interface element with comment added.
</pre>
